components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/core/ServerIdpManagementService.java

```diff
@@ -2765,7 +2765,7 @@ private DefinedByType resolveDefinedByTypeToUpdateFederatedAuthenticator(String
          If the authenticator config is present in the ApplicationAuthenticatorService list, return its type,
          if not return USER. */
         FederatedAuthenticatorConfig authenticatorConfig = ApplicationAuthenticatorService.getInstance()
-                .getFederatedAuthenticatorByName(authenticatorName);
+                .getFederatedAuthenticatorByName(authenticatorName, "test");
```

Comment:
what is this "test" ?

Reply:
Fixed with https://github.com/wso2/identity-api-server/pull/828/commits/95fe86cbe25d096bee7d8125d2cb748a2b8cc586